add2cal/add2cal.py

```diff
@@ -29,14 +34,17 @@ def __init__(
         end,
         description,
         location,
-        timezone
+        timezone,
+        attendee
```

Comment:
if this attendee field is required, do we have PRs on other repos where we use this library updated to support this? if not we need to make this an optional field with a default.

Reply:
There will be follow up PRs. It should be optional regardless. Thanks